src/compiler/README.md

```diff
@@ -112,21 +114,21 @@ pub struct CommonPreprocessedInput {
   /// multiplicative group order
   pub group_order: usize,
   /// Q_L(X): left wire selector polynomial
-  pub ql:          Poly,
+  pub ql:          usize,
```

Comment:
I think this is a polynomial only. Can be changed to `Polynomial<Lagrange, PlutoScalarField, GROUP_ORDER>`.

Refer:  https://github.com/pluto/ronkathon/blob/main/src/compiler/program.rs#L78-L97